subsys/bluetooth/host/hci_core.c

```diff
@@ -3014,6 +3014,66 @@ static void update_sec_level(struct bt_conn *conn)
 #endif /* CONFIG_BT_SMP */
 
 #if defined(CONFIG_BT_SMP) || defined(CONFIG_BT_BREDR)
+#if defined(CONFIG_BT_BREDR)
+static bool key_size_is_sufficient(u8_t key_size, bt_security_t sec_level)
+{
+	if (sec_level == 4) {
```

Comment:
We can argue about the usage of `4` here rather than the enum name. I went with the integer to make something that applies easily both to master and LTS, since we've recently renamed/deprecated the security level enum values.